x/concentrated-liquidity/grpc_query.go

```diff
@@ -121,3 +125,33 @@ func (q Querier) Params(goCtx context.Context, req *types.QueryParamsRequest) (*
 	ctx := sdk.UnwrapSDKContext(goCtx)
 	return &types.QueryParamsResponse{Params: q.Keeper.GetParams(ctx)}, nil
 }
+
+// LiquidityDepthsForRange returns liquidity depths for the given range.
+func (q Querier) LiquidityDepthsForRange(goCtx context.Context, req *types.QueryLiquidityDepthsForRangeRequest) (*types.QueryLiquidityDepthsForRangeResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// use constant pre-defined to limit range
+	maxRange := sdk.NewInt(liquidityDepthRangeQueryLimit)
```

Comment:
nit: let's define this as a package-global variable to avoid re-allocations in the constructor every time the query is called

Reply:
resolved in the most recent commit